src/mlpack/methods/pca/pca_main.cpp

```diff
@@ -10,6 +10,8 @@
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
+
+
```

Comment:
No need to add extra lines.  (That also applies a few other places later in the file.)